frontend/src/routes/Video.tsx

```diff
@@ -353,36 +328,23 @@ const Metadata: React.FC<MetadataProps> = ({ id, event }) => {
     const { t } = useTranslation();
     const user = useUser();
 
-    const metadataContainer = useRef<HTMLDListElement>(null);
     const textContainer = useRef<HTMLDivElement>(null);
 
-    const [metadataHeight, setMetadataHeight] = useState(metadataContainer.current?.scrollHeight);
-    const [buttonVisible, setButtonVisible] = useState(false);
-    const [contentExpanded, setContentExpanded] = useState(false);
-    const [linesToShow, setLinesToShow] = useState(5);
+    const [expanded, setExpanded] = useState(false);
+    const [showButton, setShowButton] = useState(false);
 
-    useEffect(() => {
-        if (metadataContainer.current) {
-            resizeObserver.observe(metadataContainer.current);
+    const resizeObserver = new ResizeObserver(() => {
+        if (textContainer.current) {
+            const textHeight = textContainer.current.scrollHeight;
+            // The creator line with margin takes 33px, the height of the container is 200px
+            // and has a top and bottom padding of 12px each.
+            setShowButton(textHeight + 33 > 200 - 12 - 12);
```

Comment:
well ok, that unfortunately doesn't work here, but my previous math doesn't make any sense either, as `200` is just the _minimum_ height of the container, meaning it can be larger than that. Whoops.